### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by shardendu930**
Added twitter logo.

### CODE REVIEW

**Comment by CandidDeer** [comment]
@Syknapse Are you okay with this change to the index (adding twitter logo)? Thx.

File: index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css" />
     <link rel="icon" type="image/png" href="favicon.png" />
     <title>Contribute To This Project</title>
+    <script src="https://kit.fontawesome.com/3c76869928.js" crossorigin="anonymous"></script>
```
Comment (Syknapse) [inline index.html:13]: We already have font awesome cdn added to the head just a few lines up.

File: index.html [hunk 2]
```diff
@@ -63,7 +64,7 @@ <h2>A project for first-time contributions</h2>
                 data-show-count="false"
                 title="Tweet this project"
               >
-                Tweet
+                <i class="fa-brands fa-twitter"></i>
```
Comment (Syknapse) [inline index.html:67]: I can't see this as an improvement. The text is more accessible for screen readers

**Review by Syknapse — COMMENTED** [review]
I really appreciate the effort, but in this case I'm going to have to reject this PR because I don't see it improving anything and instead it makes the button less accessible. I'm going to close this PR now but we would love to see a second contribution from you soon that hopefully adds some value.

**Pull request closed without merging**